scenes/game_elements/characters/npcs/talker/components/talker.gd

```diff
@@ -3,6 +3,7 @@
 @tool
 class_name Talker
 extends NPC
+## @deprecated: Instead of instantiating this class, use a [TalkBehavior] node instead.
```

Comment:
Huh, I didn't realise this was the right place to put the `@deprecated` annotation!